<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="ViewWidget" name="widget" native="true"/>
    </item>
    <item row="0" column="1">
     <widget class="QScrollBar" name="verticalScrollBar">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="singleStep">
       <number>8</number>
      </property>
      <property name="pageStep">
       <number>32</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QScrollBar" name="horizontalScrollBar">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="singleStep">
       <number>8</number>
      </property>
      <property name="pageStep">
       <number>32</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Spr"/>
    <addaction name="actionImport_s16"/>
    <addaction name="actionImport_Blk"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionRemove_Hidden_Pixels"/>
   </widget>
   <widget class="QMenu" name="menuReplace">
    <property name="title">
     <string>Replace</string>
    </property>
    <widget class="QMenu" name="menuBackground_3">
     <property name="title">
      <string>Background</string>
     </property>
     <addaction name="actionBackgroundColorMap"/>
     <addaction name="actionBackgroundBumpMap"/>
     <addaction name="actionBackgroundEffectMap"/>
     <addaction name="actionBackgroundBakedMap"/>
    </widget>
    <widget class="QMenu" name="menuForeground_2">
     <property name="title">
      <string>Foreground</string>
     </property>
     <addaction name="actionForegroundColorMap"/>
     <addaction name="actionForegroundBumpMap"/>
     <addaction name="actionForegroundEffectMap"/>
     <addaction name="actionForegroundBakedMap"/>
    </widget>
    <addaction name="menuBackground_3"/>
    <addaction name="menuForeground_2"/>
    <addaction name="actionCutouts"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_Background"/>
    <addaction name="actionShow_Foreground"/>
    <addaction name="actionShow_Cutouts"/>
    <addaction name="actionRooms"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Color_Map"/>
    <addaction name="actionShow_Bump_Map"/>
    <addaction name="actionShow_Effect_Map"/>
    <addaction name="actionShow_Baked_Map"/>
    <addaction name="separator"/>
    <addaction name="actionActual_Size"/>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuRooms">
    <property name="title">
     <string>Rooms</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionSelect"/>
    <addaction name="separator"/>
    <addaction name="actionImportRooms"/>
    <addaction name="separator"/>
    <addaction name="actionMeld"/>
    <addaction name="actionRemove_Thin_Rooms"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuReplace"/>
   <addaction name="menuRooms"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="document-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionImport_Spr">
   <property name="text">
    <string>Import Spr</string>
   </property>
  </action>
  <action name="actionImport_s16">
   <property name="text">
    <string>Import s16</string>
   </property>
  </action>
  <action name="actionImport_Blk">
   <property name="text">
    <string>Import Blk</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-undo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-redo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-cut">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="edit-delete">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionBackgroundColorMap">
   <property name="text">
    <string>Color Map</string>
   </property>
  </action>
  <action name="actionBackgroundBumpMap">
   <property name="text">
    <string>Normal Map</string>
   </property>
  </action>
  <action name="actionBackgroundEffectMap">
   <property name="text">
    <string>Effect Map</string>
   </property>
  </action>
  <action name="actionBackgroundBakedMap">
   <property name="text">
    <string>Baked Map</string>
   </property>
  </action>
  <action name="actionForegroundColorMap">
   <property name="text">
    <string>Color Map</string>
   </property>
  </action>
  <action name="actionForegroundBumpMap">
   <property name="text">
    <string>Bump Map</string>
   </property>
  </action>
  <action name="actionForegroundEffectMap">
   <property name="text">
    <string>Effect Map</string>
   </property>
  </action>
  <action name="actionForegroundBakedMap">
   <property name="text">
    <string>Baked Map</string>
   </property>
  </action>
  <action name="actionShow_Background">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Background</string>
   </property>
  </action>
  <action name="actionShow_Foreground">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Foreground</string>
   </property>
  </action>
  <action name="actionShow_Cutouts">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Cutouts</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset theme="zoom-in">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset theme="zoom-out">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionActual_Size">
   <property name="text">
    <string>Actual Size</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionImportRooms">
   <property name="text">
    <string>Import Room Map</string>
   </property>
  </action>
  <action name="actionRooms">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Rooms</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCutouts">
   <property name="text">
    <string>Cutouts</string>
   </property>
  </action>
  <action name="actionShow_Bump_Map">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Bump Map</string>
   </property>
  </action>
  <action name="actionShow_Color_Map">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Color Map</string>
   </property>
  </action>
  <action name="actionShow_Baked_Map">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Baked Map</string>
   </property>
  </action>
  <action name="actionShow_Effect_Map">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show Effect Map</string>
   </property>
  </action>
  <action name="actionRemove_Hidden_Pixels">
   <property name="text">
    <string>Remove Hidden Pixels</string>
   </property>
  </action>
  <action name="actionMeld">
   <property name="text">
    <string>Meld</string>
   </property>
  </action>
  <action name="actionRemove_Thin_Rooms">
   <property name="text">
    <string>Remove Thin Rooms</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ViewWidget</class>
   <extends>QWidget</extends>
   <header>viewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
